libs/server-sdk/src/data_interfaces/system/idata_system.hpp

```diff
@@ -21,19 +21,13 @@ class IDataSystem : public IStore {
      */
     virtual void Initialize() = 0;
 
```

Comment:
Removed Shutdown because it has no use at this time. Shutting down the `io_service` causes all of our async stuff to close, and then destructors take care of the rest.